### DISCUSSION TITLE
Where is the reporting point for Github Docs?

### DISCUSSION THREAD

**Discussion by la-png** [community/community#124874] [Other Feature Feedback, Questions, & Ideas] [answered]
I found an error in Docs. I would like to report it, but I don't know where it is. I would appreciate it if you could let me know.

For your information, it's number 2 on this page.
https://docs.github.com/ja/authentication/connecting-to-github-with-ssh/testing-your-ssh-connection?platform=linux

**Comment by davevad93** [comment]
Hi @la-png ,

At the bottom of the [docs main page](https://docs.github.com/en) you will see this: 

![git](https://github.com/community/community/assets/94308234/ca094eac-e12b-448f-8cb8-ad667a71d726)

It will redirect you to [github docs repository](https://github.com/github/docs) `fork` the repo and make a `pull request` with the fix.

**Comment by la-png** [comment]
Thanks for the introduction.
I have searched within this repository and could not find any JA data. Can you please tell me where I can put it out and submit a pull request?

**Reply by nithinlycanz** [reply]
Just with in the contribution section label and address it as an issue the maintainers might help you.

**Reply by davevad93** [reply, marked as answer]
Hey @la-png sorry I didn't know you were looking for the `ja` docs, in that case you should open an `issue` in the [github docs repository](https://github.com/github/docs) reporting the bug in the Japanese documentation and label it as `localization`.

![1](https://github.com/community/community/assets/94308234/ef5bf1aa-252e-43b1-bfcc-4f799e79d350)
![2](https://github.com/community/community/assets/94308234/d1c01851-a5fe-4bf1-921f-4cdce813d192)

**Reply by la-png** [reply]
Thanks, I have submitted the issue.

**Comment by nithinlycanz** [comment]
@la-png feel free to mark answer
